src/DynamoCore/Configuration/DebugModes.cs

```diff
@@ -43,34 +43,35 @@ private static void AddDebugMode(string name, string description)
         }
         private static void RegisterDebugModes()
         {
-            // Add new debug modes here!!
-            //
-            // Example:
-            // AddDebugMode("TestDebugMode", "Enabe/disable TestDebugMode.");
+            // Register app wide new debug modes here.
+            AddDebugMode("Python3DebugMode", "Enabe/disable Python3DebugMode.");
```

Comment:
@pinzart I have registered a new Debug mode for our Python3 feature and added a debug.config file in Dynamo root. Curious how to reference it?

Reply:
You can use if (DebugModes.Enabled("Python3DebugMode") == true) to check if your debug mode is enabled or not.

As a side note...I put the DebugModes class in DynamoCore...in the hopes that all other Projects in Dynamo can include the Dynamo.Configuration namespace ...in order to have access to DebugModes.

Reply:
as for the Description section of the DebugMode...It only helps to figure out what the feature is, behind the debug mode. This is only visible in the DebugModes UI.
You can leave out the fact that it is a debug mode ....Like Enable/Disable Python 3 Engine....or something like that..
Anyway...it's a description...you can put anything you want theere...

Reply:
@pinzart Adding the name space may not help since I am using the reference in a different dll here, I had to add internally visible to key word. See my changes